Geometry_Engine/Create/BoundingBox.cs

```diff
@@ -51,6 +53,19 @@ public static BoundingBox BoundingBox(Point centre, Vector extent)
             };
         }
 
+        /***************************************************/
+
+        [Description("Create a BoundingBox using the properties of a cuboid.")]
+        [Input("globallyAlignedCuboid", "Cuboid that is assumed to be globally aligned.")]
```

Comment:
It could be worth mentioning that globally aligned means centre == origin && aligned axes (particularly the former is not clear from the current description).